src/library_html5.js

```diff
@@ -175,7 +175,7 @@ var LibraryJSEvents = {
     },
 
 #if USE_PTHREADS
-#if MINIMAL_RUNTIME
+#if MINIMAL_RUNTIME && !WASM_BACKEND
```

Comment:
why is the wasm backend relevant here?

Reply:
In asm.js backend the stackSave, stackRestore and stackAlloc functions are generated from JS code with an `__asm: true` directive. in Wasm backend they are part of the wasm module. Deps handling works differently between those two.